### DISCUSSION TITLE
Is it possible to disable the self update feature for the prebuilt binaries?

### DISCUSSION THREAD

**Discussion by SinTan1729** [topgrade-rs/topgrade#333] [Q&A] [answered]
Is it possible (maybe using `topgrade.toml`) to disable the self-update feature for the prebuilt binaries?

**Comment by s34m** [comment, marked as answer]
There is the TOPGRADE_NO_SELF_UPGRADE environment flag you can set to false to disable it. Or else you can compile the binary without the self-update flag.

**Reply by SinTan1729** [reply]
What does the `self_update` option under `[composer]` do in the config file?
Also, setting that environmental variable to `true` and `false` both stops self update from running, which is kinda weird imo.

**Reply by s34m** [reply]
1. It just allows or stops the composer tool from updating itself
2. What😂, will fix that

**Comment by etrigan63** [comment]
Setting `self_update = false` in `topgrade.toml` does nothing.

**Reply by SinTan1729** [reply]
The PR #388 adds a flag `no_self_update`. Setting it to `true` disables self update.
